django/contrib/admin/sites.py

```diff
@@ -302,14 +304,18 @@ def each_context(self, request):
         """
         script_name = request.META['SCRIPT_NAME']
         site_url = script_name if self.site_url == '/' and script_name else self.site_url
-        return {
+        context = {
             'site_title': self.site_title,
             'site_header': self.site_header,
             'site_url': site_url,
             'has_permission': self.has_permission(request),
             'available_apps': self.get_app_list(request),
             'is_popup': False,
+            'nav_sidebar_enabled': self.enable_nav_sidebar,
         }
+        if self.enable_nav_sidebar is True:
+            context['nav_app_list'] = self.get_app_list(request)
```

Comment:
There is variable called `available_apps` that keeps app list and available across whole admin ;)